spec/controllers/v0/benefits_claims_controller_spec.rb

```diff
@@ -135,14 +135,16 @@
                 { message_type: 'lh.cst.evidence_requests',
                   claim_id: '600383363',
                   tracked_item_id: 395_084,
-                  tracked_item_type: 'Request 1' })
+                  tracked_item_type: 'Request 1',
+                  tracked_item_status: 'NEEDED_FROM_YOU' })
```

Comment:
It doesn't really matter what the value of this status is, but I changed it to be more reflective of what it seems like this data would be.